bot/start.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# startup script for wuestbot on gunicorn
+
+# Project Name
+NAME="wuestbot"
+
+# Host to listen on
+HOST=127.0.0.1
+
+# Port to listen on
+PORT=5000
+
+# Number of gunicorn Workers
+NUM_WORKERS=2
+
+# Databank directory
+DB_DIR=app/database
+
+# start venv
+source venv/bin/activate
+
+# create database
+source $DB_DIR/db_create.py
+
+# startup server
+exec gunicorn $NAME:app \
+    -b $HOST:$PORT \
+    -w $NUM_WORKERS \
```

Comment:
We also need to pass the API_TOKEN via a variable before the bot is started.

Reply:
i'd think it's more sensible to do this in the `.env` file and keep the token secret and retrieve it via `load_dotenv()`